push_ios/ios/Classes/UserNotificationCenterDelegateHandlers.swift

```diff
@@ -60,4 +58,40 @@ class UserNotificationCenterDelegateHandlers: NSObject, UNUserNotificationCenter
             originalDelegate.userNotificationCenter?(center, openSettingsFor: notification)
         }
     }
+
+    // Parses userInfo to get presentation options. If some present value is not found, it is  considered as true
```

Comment:
I wonder why we should default to true?

Reply:
We can change. I made so because those parameters are also true by default in flutter_local_notifications. However, they have an ability to set other default values in initialization settings.

Reply:
Great, thanks